README.md

```diff
@@ -110,6 +110,9 @@ For each environment you specify in your configuration file, **astilectron-bundl
 - `AppName`:  filled with the configuration app name
 - `BuiltAt`: filled with the date the build has been done at
 
+If you need to add more flags yourself, like for a version number, add something
+like this to your `astilectron-bundler` command: `-ldflags main.version=xyzzy`
```

Comment:
I think we need to provide a way to indicate `-X` or `-h` ldflags for instance in this flag.

Therefore I'd rather you create a new struct that implements the `flag.Value` interface so that it can be passed to `flag.Val()` and that mimics the `bundler.ldflags` struct.

As a result the command could be for instance:

`-ldflags X:main.version=xyzzy,main.test=rere -ldflags s:foo=bar`

Or something similar.